<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QListView" name="listView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListView::item:!selected {
    border: 1px solid gray;
    border-left-color: transparent;
}

QListView::item:!selected:hover {
    background: #f3f3f3;
    color: #252424;
}</string>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QTreeView" name="table">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="indentation">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="indexLabel">
             <property name="text">
              <string>Index:</string>
             </property>
             <property name="buddy">
              <cstring>spinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>200</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Operation &amp;history</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QUndoView" name="undoView">
           <property name="emptyLabel">
            <string>Initial state</string>
           </property>
           <property name="cleanIcon">
            <iconset resource="images.qrc">
             <normaloff>:/svg/icons/save.svg</normaloff>
             <selectedoff>:/svg/icons/save_dark.svg</selectedoff>
             <selectedon>:/svg/icons/save_dark.svg</selectedon>:/svg/icons/save.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnEdit">
            <property name="text">
             <string>Edit</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/svg/icons/edit.svg</normaloff>:/svg/icons/edit.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConfirmOp">
            <property name="text">
             <string>Confirm operation</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/svg/icons/greenCheckmark.svg</normaloff>:/svg/icons/greenCheckmark.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDiscardOp">
            <property name="text">
             <string>Discard operation</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/svg/icons/resultIncorrect.svg</normaloff>:/svg/icons/resultIncorrect.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="HexEdit" name="hexEdit">
          <property name="font">
           <font>
            <family>Monospace</family>
            <pointsize>11</pointsize>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSaveChanges"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveHistory"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionMountDummy"/>
    <addaction name="actionCheckConsistency"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuDefaultChar">
     <property name="title">
      <string>&amp;Default character</string>
     </property>
     <addaction name="actionCharPoint"/>
     <addaction name="actionCharUnderscore"/>
     <addaction name="actionCharSpace"/>
    </widget>
    <addaction name="actionViewASCII"/>
    <addaction name="menuDefaultChar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEditor"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSaveChanges"/>
   <addaction name="separator"/>
   <addaction name="actionMountDummy"/>
   <addaction name="actionCheckConsistency"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/svg/icons/incompleteFolder.svg</normaloff>:/svg/icons/incompleteFolder.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionViewASCII">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;View ASCII</string>
   </property>
  </action>
  <action name="actionSaveChanges">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/svg/icons/save.svg</normaloff>:/svg/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionTrackChanges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tracking mode</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSaveHistory">
   <property name="text">
    <string>Save &amp;history</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/svg/icons/addAny.svg</normaloff>:/svg/icons/addAny.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Create...</string>
   </property>
   <property name="toolTip">
    <string>Create</string>
   </property>
  </action>
  <action name="actionMount">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/svg/icons/mount.svg</normaloff>:/svg/icons/mount.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Mount...</string>
   </property>
   <property name="toolTip">
    <string>Mount at directory</string>
   </property>
  </action>
  <action name="actionCheckConsistency">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/svg/icons/magicResolveToolbar.svg</normaloff>:/svg/icons/magicResolveToolbar.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Check errors</string>
   </property>
   <property name="toolTip">
    <string>Check filesystem consistency</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>C&amp;lose</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/svg/icons/qt.svg</normaloff>:/svg/icons/qt.svg</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionUnmount">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/svg/icons/ejectFolder.svg</normaloff>:/svg/icons/ejectFolder.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Unmount</string>
   </property>
   <property name="toolTip">
    <string>Unmount from directory</string>
   </property>
  </action>
  <action name="actionMountDummy">
   <property name="text">
    <string>MountDummy</string>
   </property>
  </action>
  <actiongroup name="charGroupTool">
   <action name="actionCharPoint">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&amp;Point (.)</string>
    </property>
   </action>
   <action name="actionCharUnderscore">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&amp;Underscore (_)</string>
    </property>
   </action>
   <action name="actionCharSpace">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&amp;Space ( )</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexEdit</class>
   <extends>QFrame</extends>
   <header location="global">hexedit/hexedit.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
